latex/epoch.tex

```diff
@@ -37,11 +39,21 @@
 %% Figure - Epoch Abstract Types
 %%
 \begin{figure}[htb]
-  \emph{Abstract types}
+  \emph{Derived types}
   %
   \begin{equation*}
-    \begin{array}{r@{~\in~}ll}
-      \var{production} & \Production & \text{blocks produced in the epoch}\\
+    \begin{array}{r@{~\in~}lr}
+      hk \mapsto n & \Production & \text{blocks produced in the epoch}
+    \end{array}
```

Comment:
is Production : VKey \to N? Or HashKey \to N?
Also Production is just the number of blocks associated to this key in this epoch?
Is it not used at all anywhere, not even the stakeDistr calculation?

Reply:
yes, it's not used yet, but it is about to be used for the rewards calculation. I probably should have not included this in this PR.